flag_engine/segments/models.py

```diff
@@ -68,6 +69,12 @@ def evaluate_modulo(self, trait_value: typing.Union[str, int, float, bool]) -> b
             return False
         return trait_value % divisor == remainder
 
+    def evaluate_in(self, trait_value) -> bool:
+        try:
+            return str(trait_value) in self.value.split(",")
+        except ValueError:
```

Comment:
@EdsnLoor I don't think there is any scenario where the logic above can throw a `ValueError` - all data types should be able to be converted to strings. There is one possible exception that we should catch though. Related to the `self.value.split(",")` logic.